emodpy_typhoid/interventions/typhoid_vaccine.py

```diff
@@ -64,6 +65,16 @@ def new_vax( camp, efficacy=0.82, mode="Acquisition", constant_period=0, decay_c
     else:
         raise ValueError( f"mode {mode} not recognized. Options are: 'Acquisition', 'Transmission', or 'All'." )
 
+    # replace: DAD=1, EIR=1
+    # combine: DAD=1, EIR=0
+    intervention.Dont_Allow_Duplicates = 1
+    if deduplication_policy == "replace":
+        intervention.Enable_Intervention_Replacement = 1
+    elif deduplication_policy == "combine":
+        intervention.Enable_Intervention_Replacement = 0
+    else:
+        raise ValueError( f"duplication_policy needs to be 'replace' or 'combine', not '{duplication_policy}'." )
```

Comment:
{duplication_policy} should be {deduplication_policy}

Reply:
Good catch!